src/pages/profile/Profile.tsx

```diff
@@ -24,172 +24,163 @@ const initialState = {
 
-                                    <img src="/logo192.png" alt="other" />
-                                </FormGroup>
-                            </Col>
-                            <Col md={6} style={{ margin: "auto" }}>
-                                <FormGroup>
-                                    <button style={{ width: "100%" }}>Change Image</button>
-                                </FormGroup>
-                            </Col>
-                        </Row>
-                        <Row>
-                            <Col md={6}>
-                                <FormGroup>
-                                    <Label>Email Address</Label>
+                                <img src="/logo192.png" alt="other" />
+                            </FormGroup>
+                        </Col>
+                        <Col md={6} style={{ margin: "auto" }}>
+                            <FormGroup>
+                                <button style={{ width: "100%" }}>Change Image</button>
+                            </FormGroup>
+                        </Col>
+                    </Row>
+                    <Row>
+                        <Col md={6}>
+                            <FormGroup>
+                                <Label>Email Address</Label>
 
-                                    <TextInput label="Email" name="emailAddress" type="text" />
-                                </FormGroup>
-                            </Col>
-                            <Col md={6}>
-                                <FormGroup>
-                                    <Label>Gender</Label>
-                                    <SelectInput label="Gender" name="gender">
-                                        <option value="MALE">Male</option>
-                                        <option defaultChecked value="FEMALE">
-                                            Female
-                                        </option>
-                                    </SelectInput>
-                                </FormGroup>
-                            </Col>
-                        </Row>
-                        <Row>
-                            <Col md={6}>
-                                <FormGroup>
-                                    <Label>Language</Label>
-                                    <SelectInput label="Language" name="language">
-                                        <option value="ENGLISH">English</option>
-                                        {/* <option defaultChecked value="SPANISH">
+                                <TextInput label="Email" name="emailAddress" type="text" />
+                            </FormGroup>
+                        </Col>
+                        <Col md={6}>
+                            <FormGroup>
+                                <Label>Gender</Label>
+                                <SelectInput label="Gender" name="gender">
+                                    <option value="MALE">Male</option>
```

Comment:
Set these values to be the enum value instead.

Reply:
Apply throughout. The reason to do this is so you have better type safety and insurance that you didn't do accidental misspelling.